release/readme.in.html

```diff
@@ -1,488 +1,595 @@
2M1Q3z1C/YDYKEcq0DYkChnkRfbIR0d7AG7QL4GQxAkUOZQJI8YgLdUCLwQL69yexIEQQIlCIIgSKAEQRAkUIIgCBIoQRAECRSrurjcQqtFc6z24qILXLKCazl0/qsh1OWhUZNRl4MaUetRskCxSRvSxH9YBEEQtR6lCRSJJ+5c+d5nQBAEQQIlexIEQQJVcOReeaf4+s1XyWvBVwkQVaIOlTIRvMj/yDazb9X26RMEQQLF4L1S5z2Z9QRKFLpU9kDoS3lf1SlQgiBIoFhzr9RTFCegrKZEkE0KM1LZVsC/CSJKqu3TJwiCBIodS5UuULE/35l/CuIpAyUI4l8oUOz3rGKB8qnNUrX46sXLZw/vP3/0ELe745UPnz6PyL0TnFEQnH7zjweP/xUCJQiCBFrZGz9lxMko3ah/5WTGHXA6sWi6x5BuHr20ffvoHR/ZL2q9yd3keLTuC7usMtlGYuQtMTrc0iEk++8n1S9QgiBIoLjWqLIFevv+HyHZcT4pZz2Sz57JjEwsSLvz4E++4H47JenIounb2zTa3ew7H/W6Z7SaBndoEaLdIriDaqBWk4v6atcczCbaHZLM2CaxjlSxjvreIbbgwQtYt7IFSuD+pHiA2okjh/t27VJaAEGQQNXVKvUULc/t/2FdD8mqDpLNHeo4DmzvMq2z65xubnOH+ZrsjvWVbrexU6+zU/XbAG3VMH0NqYFmuJ5amL56mE6rMAMN1ITrtQpv19SxS5cvlhyQ2MZIzMO7elxm8i2/QGMiIq5euSyHuJho2eDr164B2YDkpEQ8VwMPlB8xYICg8xXLl7GYZQsW4LA2gafyso9i7NChpQQQBAlUp5IF+uEvbSVrdJq4j/LNDfr7+cPikuJXr4vuv3h66U7GynGdNtb91LNNIxhTCl3CmwZqIe1UUY7urQN7ogagEKvbpO8CO4ldnMpmqVV0IRv2l1+geBQtLCAHPPeinMF48O/CWTNlO7fYsIE1rV+5ggSqTAiCBPrzqp4SGwOn9GN8qzz768waQ4v6X/prvdFlKOwJdFuFtmt8adGMwtiY6AuBezQbhuipwZ4gVkd1wVRjiV28iqXUPDjzWHL+4aTcs6k3n7148V4CPX3iRKn4e3uJgzF0ZZw5eSIhLo47FI8aXzhrFgmUICodEqj5mT2Sde3PFETLLrpfDTi6vv7nHq0bReirQ53hEKXum2H79b2OaN10PGp4j8E2qt+c12mJJgg0Xkd1+ow1EocEiVWEyuqTEtMjn5geVTE93u9gXPkFilF2RYInjBgeHhqKJoACCZQgKh0S6KNnTwwcpy4J2crt+erZU6cBHbc0+xaJJ+QYCoeC9i0urzcpTIqP3eNo3VXXTvV7r7aNw9k0qEGrUH0NtVWeH9rGfmwVIbGJeoNttIpl5ADftCoTKBjUq2dmRjpaQQ8DfYUF2r9rl3nTp+GF9pYWS+bNHdKrFyrFvp47bdrk0aPKapo4aiSvwcwsamZMnIByp3Za0yeMt968ycbMDJ336dK5rNNAk/GSXxBmZ2GBnBpnJUeg+m1a4x3N169D/JypU7rpaCOg/HTUaovTXrFs6RYb6w2rVk4ZM7qrdgfeKv4p8HZzp041X78e8auNjIb164tKBeIJonIFWtmniNnK3N9vGDjPSPgtkwk0PzJsY+Mvj7ZrKu2o/h9FqgUaqLnrqe3TqO/a8udjbRsF6rbCghLEigw0QbvJ8omLP7aJUbGOlFhKJRbSjy2kEjOpyuawXUl3qlKgAEtJaAWLZs9WQKB49KbTju0IFhB49gw3MiM+Nhb1GWlp4k6wooWmq5cv8xpMMrB+4MHszEwUOBnp6RCKuJPd27djLkJwGudPn+7duZNYoMsXLsTb/b/g/Hys0SOmPEwZOwbLdIL3wrsb/bIYreKfYtbkSZHScMHbHXD+VVdTQ/F4gqiJGSiTZvrd/PknrQof/oZyiIO5ZYMvgvTUwt8CgSLBxBA+5E0Z3tQAGNeH6rUMbdcU341HzfzybeKpYh3Rwimhx8Grfbyv9vVKHeCdmvnX0yoW6EGXA2gFsJICAsV8KwsWg9V/JGUVEmjZCDJZpGllRSYnJgzp01sgUDFMUkvmz0OYfJAOI7KsTlYZGYp/CqzUlRpsa26OMIXjCaKmChRfvz/8O6bgSvHr136LpzuqfhOupwlLYtn9bRKqHqqjFtqueYhWs6C2TUK03ngzZmCX/M2rpy63kMzbDXtKrKQ/b4+79ZDvAP3nryoWaHhwMFrBmKFD3legGLeySGBvZYnDGRMmbLez49MCGJBWXKAoIw/FGBZ2Y5EA2zl5MJzFgz1cXRGMUTBO48qlS6wS3hQLVBoSgn1aw/v3w0iZ7/Q6d+oUwuQTcOwYCw4NDsIp4XPDxi882p5VBl+4UOpPcTklxWrTRnxESxfMP+rnx+vnz5iucDxB1DyB8ks2ufTcp4zY0+L7/6SZsOeb9feY4b2T501KW7k0d4f17SPeDy4lFj9+fLvwZmPD3R8scfvQNkrFXNrPO5V1o/A2JmxXKhVIsDwCXWtqwvpJT03Va635vgKFH/H/PNi8do1s/T6n3ayHHQ72FRToxXPnOnVoz+shaFaPl4h/B8Cest3CjKwen0B3XR1ZgWIA3qtTRx6JFDU/L4/ND7DPQQ7eBw/i9AS78THfio8dPaAfnncjjI/uMernwbptWp8NOMmaMIWCGsXiCaImClSovGPGC5ybfR/RUR3LR5j9DDNoFaKj9j8JsYLA6F32v+hrfzDJXGX1CSwZScwiLCJvoDUsJyHvzwJEKXEf6FYbG3HwyIEDGOOHD0Ne43PoEI/3dHNT4io8xtesB7xFBQUqmFUEPK/EMg5bYmLuw8sxISsIXmm43MHaCiNfrMkwgZY1Fg46f541De7dC4cKcPLoUdYD1oL4TyH+RcKAfFlTYnw8DhWLJ4gaKVAmO+7HqL07HZt+J2W7l97ulseYvdD3EL+/yF95WYcXTrOt/9nA7oM/mrlNxTxEY
```

Comment:
Looks like a full list of defects needs to somehow get posted on the releases page. I guess that's the next step.